source/_posts/en/tutorial/component.md

```diff
@@ -70,28 +68,28 @@ var MyApp = san.defineComponent({
 ```
 
 
-lifecycle
+Lifecycle
 ------
 
-San 的组件是 HTML 元素扩展的风格，所以其生命周期与 WebComponents 相符合。
+San components use HTML-based template syntax, so their lifecycle is consistent with WebComponents.
```

Comment:
syntax, so their lifecycle is consistent with WebComponents
syntax in order too keep their lifecycles consistent with WebComponents